features/currencyexchange/database/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.serialization)
+    alias(libs.plugins.sqldelight)
+}
+
+group = "com.mindera.currencyexchange.database"
+version = "1.0.0"
+
+kotlin {
+    androidTarget()
+    jvm()
+    iOS {
+        iosX64()
+        iosArm64()
+        iosSimulatorArm64()
+    }
+
+    sourceSets {
+        all {
+            languageSettings {
+                optIn("kotlin.experimental.ExperimentalObjCRefinement")
+            }
+        }
+        val commonMain by getting {
+            dependencies {
+                implementation(libs.ktor.core)
+                implementation(libs.ktor.content.negotiation)
+                implementation(libs.ktor.logging)
+                implementation(libs.ktor.serialization)
+                implementation(libs.kotlinx.serialization)
+                implementation(projects.features.currencyexchange.data.remote.mindera.rest.schema)
```

Comment:
Indentation.

Reply:
[formatting](https://github.com/Mindera/Android-KMP-Template/pull/9/commits/b02611b1dda41dda34e9bbbfe36ba5faddb738bc) I hope I did it right